LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,8 @@
 		535A6CD61E786B92008472D3 /* Trip.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535A6CD51E786B92008472D3 /* Trip.swift */; };
 		535A6CD81E788F5A008472D3 /* SettingsController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535A6CD71E788F5A008472D3 /* SettingsController.swift */; };
 		535C15B11E785CFF00171705 /* LocationManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535C15B01E785CFF00171705 /* LocationManager.swift */; };
+		53F135271E940A1F0014DFF5 /* PinInput.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53F135261E940A1F0014DFF5 /* PinInput.swift */; };
+		53F135291E940A810014DFF5 /* GUIDInputViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53F135281E940A810014DFF5 /* GUIDInputViewController.swift */; };
```

Comment:
If `.DS_Store` was in your `.gitignore`, there would not be any in this PR. :)

And I defer to your expertise on the project file. Whatever makes sense in Swift land.